cfbs/utils.py

```diff
@@ -56,6 +57,8 @@ def touch(path: str):
 def rm(path: str, missing_ok=False):
     if not missing_ok:
         assert os.path.exists(path)
+    if missing_ok and not os.path.exists(path):
+        log.warning("delete: path '%s' does not exist, skipping" % path)
```

Comment:
I would not put this warning here, because we might use it places where we expect the operation to fail. However you can change the `rm()` function to return `True` or `False` based on the success of the operation. Then you can make decisions based on this condition elsewhere.

Reply:
Sure, that makes sense. I can refactor back to the "delete" step.